src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form>
+      <fieldset class="persInf">
+
+        <legend>Personal information</legend>
+        Surname:
+        <input
+        class="input"
+        type="text"
+        name = "lastname"
+        autocomplete="off"
+        required
+        >
```

Comment:
please check the spaces. and it would be great if you take a look on the checklist to be sure https://github.com/mate-academy/layout_html-form/blob/master/checklist.md
```suggestion
        <input
          class="input"
          type="text"
          name="lastname"
          autocomplete="off"
          required
        >
```